pkg/alertmanager/multitenant.go

```diff
@@ -479,10 +483,16 @@ func (am *MultitenantAlertmanager) ServeHTTP(w http.ResponseWriter, req *http.Re
 	userAM, ok := am.alertmanagers[userID]
 	am.alertmanagersMtx.Unlock()
 
-	if !ok || !userAM.IsActive() {
```

Comment:
Ideally, what I'd like to do here is use the fallback configuration we support and start an Alertmanager instance. Then, we can let you that your Alertmanager instance is warming up. 

I believe this is the only component without a UI, as this is a pattern we've not seen before.

My question here, is that a step in the right direction? Is that something we'd like to include in Cortex? Authentication/Authorization is not a concern of Cortex, under that assumption, I'd expect that any user trying to access this URL should already have a manager starting - using the fallback configuration.

Reply:
Hrm, why do we have the fallback configuration for if its not being used in this way?

Reply:
It triggers when you have an empty configuration:

https://github.com/cortexproject/cortex/blob/e36e2ae2c08fd81cb7f8d0d0f8cba1357caa5d90/pkg/alertmanager/multitenant.go#L401-L404